### PR TITLE
[res] PyTorchExamples and add op

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce res/PyTorchExamples and add Op example
- running ptem.py will generate .pth file, .onnx file, .TF saved_model
and .tflite file

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #4830, from draft  #4799

File: res/PyTorchExamples/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# PyTorch examples
+
+Python example to convert PyTorch to ONNX/TF/tflite models
+
+## Package versions
+
+- Python 3.X
+- torch==1.7.0
+- onnx=1.7.0
```
Comment (mhs4670go) [inline res/PyTorchExamples/README.md:9]:
```suggestion
- onnx==1.7.0
```

Comment (mhs4670go) [reply res/PyTorchExamples/README.md:9]: And, FYI, `pytorch` 1.7.0 can't be installed using pip in python 3.5.2. When I tried to install it in 3.6.9, it works.

**Comment by mhs4670go** [comment]
@seanshpark (optional) How about adding +x permission to `ptem.py`?

**Review by mhs4670go — DISMISSED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**